NSS-Wacky-Eels/BangazonSite
# Af ticket eight

## Link to Ticket
Closes # [Link](https://github.com/NSS-Enthusiastic-Camels/BangazonAPI/issues/8)

## Description of Proposed Changes
- Created MVC for PaymentTypes

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <af-ticketEight>
Login to program
User should see "Settings"
Click into "Settings" on NavBar and then try to create a new payment type
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* PaymentTypeModel, PaymentTypeModelController, and PaymentTypes Create View

## Mentions @username

@alexfont321
@priyankagargc27
@KaylaReid
@thatmikeparrish
@Akeen11

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
